specification/databricks/resource-manager/Microsoft.Databricks/stable/2018-04-01/databricks.json

```diff
@@ -866,6 +866,14 @@
         "requireInfrastructureEncryption": {
           "$ref": "#/definitions/WorkspaceCustomBooleanParameter",
           "description": "A boolean indicating whether or not the DBFS root file system will be enabled with secondary layer of encryption with platform managed keys for data at rest."
+        },
+        "storageAccountName": {
+          "$ref": "#/definitions/WorkspaceCustomStringParameter",
+          "description": "Name of the default DBFS storage account name."
+        },
```

Comment:
instead of  $ref , can we explitly set "type":"string". otherwise documentation returns "The value which should be used for this field." instead of "Name of the default DBFS storage account name."

Reply:
"storageAccountName":
{
  "type": "string",
  "description": "Default DBFS storage account name"
}

Reply:
remove "Name of the". Leave it has Default DBFS storage account name

Reply:
I fixed the description but not $ref, it is similar to other parameters can't change only for this parameter.